engine/src/main/java/org/terasology/rendering/dag/nodes/RefractiveReflectiveBlocksNode.java

```diff
@@ -178,7 +180,11 @@ public RefractiveReflectiveBlocksNode(Context context) {
         addDesiredStateChange(new SetInputTexture2D(textureSlot++, "engine:waterNormal", CHUNK_MATERIAL_URN, "textureWaterNormal"));
         addDesiredStateChange(new SetInputTexture2D(textureSlot++, "engine:waterNormalAlt", CHUNK_MATERIAL_URN, "textureWaterNormalAlt"));
         addDesiredStateChange(new SetInputTextureFromFbo(textureSlot++, REFLECTED_FBO_URI, ColorTexture, displayResolutionDependentFBOs, CHUNK_MATERIAL_URN, "textureWaterReflection"));
-        addDesiredStateChange(new SetInputTextureFromFbo(textureSlot++, lastUpdatedGBuffer, ColorTexture, displayResolutionDependentFBOs, CHUNK_MATERIAL_URN, "texSceneOpaque"));
+        //addDesiredStateChange(new SetInputTextureFromFbo(textureSlot++, lastUpdatedGBuffer, ColorTexture, displayResolutionDependentFBOs, CHUNK_MATERIAL_URN, "texSceneOpaque"));
```

Comment:
This was just for experimentation, since I was trying to write depth values to the color attachment.

Reply:
I can see that. This is a bit of a "philosophical" issue: what I am aiming for is Nodes that are as "single-operation" as possible (when reasonable), so that they can be combined in the graph in multiple ways.